src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1258,14 +1248,23 @@ BlankNode Visitor::visitTypesafe(Parser::BlankNodeContext* ctx) {
 
 // ____________________________________________________________________________________
 template <typename Out, typename FirstContext, typename... Context>
-Out SparqlQleverVisitor::visitAlternative(FirstContext ctx, Context... ctxs) {
+Out Visitor::visitAlternative(FirstContext ctx, Context... ctxs) {
   if (ctx)
     return {visitTypesafe(ctx)};
   else
     return visitAlternative<Out>(ctxs...);
 }
```

Comment:
For safety reasons use `{}` for `if () {...} else {...}` statements, even if there is only on statement.
On the other hand there is a redundant `{}` around the `visitTypesafe`.

Reply:
This braces here are not redundant at least at the moment. Currently they are needed for the conversion from `SparqlValues` to `Values` (a struct containt a `SparqlValues` field and with default values for all other fields) in Line 194. This seems to only be possible explicitly.